js/storage.js

```diff
@@ -0,0 +1,24 @@
+const formData = {
+  title: '',
+  author: '',
+};
```

Comment:
Great job @mrbhi 🤝

- [ ] Kindly make sure the data entered is saved in the local storage. For your case, only data that is being entered into the input fields reflects in the local storage but then when the add button is clicked, the data doesn't appear in the local storage.

Reply:
Thanks @1GORDON